boards/arm64/xenvm/doc/index.rst

```diff
@@ -88,13 +77,17 @@ configuration would not boot on your hardware. In this case you need to update
 configuration by altering device tree and Kconfig options. This will be covered
 in detail in next section.
 
-No Xen-specific features are supported at the moment. This includes:
+Most of Xen-specific features are supported at the moment. This includes:
```

Comment:
This changes looks weird.
Most of the features listed after are under development or not supported at all (xenbus for example).
More then that is is incoherent with the next chapter saying "now only following feature are supported".
Could this change be a mistake ?

Reply:
Oh, sorry, agree. I missed "not" and context was broken. Of course, these feature are NOT supported yet.

Thank you for careful review!